src/nvim/ex_docmd.c

```diff
@@ -9532,3 +9543,25 @@ static void ex_terminal(exarg_T *eap)
     xfree(name);
   }
 }
+
+/// is_live()
+/// Returns true if cmd corresponds
+/// to a live command.
+/// At the moment, only substitute has a live command.
+
+bool is_live(char_u *cmd_live)
+{
+  exarg_T ea;
+  ea.cmd = cmd_live;
+
+  // parse the command line
+  if (ea.cmd != NULL) {
+    ea.cmd = skip_range(ea.cmd, NULL);
+    if (*ea.cmd == '*') {
+      ea.cmd = skipwhite(ea.cmd + 1);
+    }
+    find_command(&ea, NULL);
+  }
+
+  return (ea.cmdidx == CMD_substitute);
```

Comment:
Use of uninitialized value if `cmd_live` is NULL. This should be under `if`, `return false` under `else`.